doc/source/migrations/index.rst

```diff
@@ -11,6 +11,13 @@ Version ``v8.2``
 ----------------
 **New Features:**
 
+- Added a new action named ``ansys/actions/hk-automerge-prs``. This action allows maintainers to auto-approve and merge
+  ``dependabot`` PRs and ``pre-commit.ci`` PRs. It is recommended to add the action at the end of the workflow, once all the
+  stages have exited successfully. That way, in case a repository has failing stages, it is not run. This action will run in case
```

Comment:
```suggestion
  stages have finished successfully. That way, in case a repository has failing stages, it is not run. This action will run in case
```